test/api/v3/integration/dataexport/GET-export_userdata.xml.test.js

```diff
@@ -36,7 +36,7 @@ describe('GET /export/userdata.xml', () => {
     expect(res.tasks.dailys[0]._id).to.equal(tasks[1]._id);
     expect(res.tasks.rewards.length).to.equal(2);
     expect(res.tasks.rewards[0]._id).to.equal(tasks[2]._id);
-    expect(res.tasks.todos.length).to.equal(3);
+    expect(res.tasks.todos.length).to.equal(6);
     expect(res.tasks.todos[1]._id).to.equal(tasks[3]._id);
```

Comment:
[This test failure](https://travis-ci.org/HabitRPG/habitrpg/builds/151077184#L10000) is due to this assertion.